scrabbler/scrabbler.py

```diff
@@ -475,6 +480,9 @@ def cross_set(self, direction):
     def set_cross_set(self, direction, new_set):
         self._cross_set[direction] = new_set
 
+    def get_tile(self):
```

Comment:
It looks like a getter for `self._tile` already exists https://github.com/jfoley-yw/scrabble/blob/fad4e28b3fe69c021ee0df81b640ed9210e2bb35/scrabbler/scrabbler.py#L463.